android-refimpl-app/app/build.gradle

```diff
@@ -19,7 +19,8 @@
 buildscript {
     repositories {
         maven {
-            url 'https://maven.google.com/'
+            //url 'https://maven.google.com/'
+            url 'https://dl.google.com/dl/android/maven2/'
```

Comment:
Maybe it can be discard. Actually https://maven.google.com/ will be redirected to https://dl.google.com/dl/android/maven2/. But former is not available in China, but the latter is.

Reply:
it needs to work on the f-droid build server

Reply:
Ok, just discard this particular change. This alternation simply was for the chinese great fire wall that blocks maven.google.com.